swagger.json

```diff
@@ -3106,6 +3276,47 @@
                 }
             }
         },
+        "UnassignedDrivingSegment": {
+            "type": "object",
+            "description": "A driving segment that has no associated driver.",
+            "properties": {
+                "segment_id": {
```

Comment:
Should this be `id` or `segment_id`?

Reply:
Hey, good point, I'll change this to `id` on the `UnassignedDrivingSegment` object.

Reply:
done